tests/WE2E/run_WE2E_tests.py

```diff
@@ -8,7 +8,7 @@
 from textwrap import dedent
 from datetime import datetime
 
-sys.path.append("../../ush")
+sys.path.insert(0, "../../ush")
```

Comment:
Per [this StackOverflow comment](https://stackoverflow.com/a/10097543), it seems like it's better practice to use 1 rather than 0 here (sys.path[0] is the current working directory).